Jenkinsfile

```diff
@@ -364,33 +337,23 @@ pipeline {
             name 'DB'
             values 'postgresql_96', 'mariadb_103'
           }
-          axis {
-            name 'PROFILE'
-            values 'engine-unit', 'engine-unit-authorizations', 'webapps-unit', 'webapps-unit-authorizations'
-          }
         }
         when {
-          anyOf {
-            branch 'pipeline-master';
-            allOf {
-              changeRequest();
-//              expression {
-//                withLabels("all-db") || withDbLabel(env.DB)
-//              }
-            }
+          expression {
+            withLabels("all-db") || withDbLabel(env.DB)
           }
+          beforeAgent true
         }
         agent {
           kubernetes {
             yaml getDbAgent(env.DB)
           }
         }
         stages {
-          stage('UNIT test') {
+          stage("engine-UNIT") {
             steps {
-              echo("UNIT DB Test Stage: ${env.PROFILE}-${env.DB}")
               withMaven(jdk: 'jdk-8-latest', maven: 'maven-3.2-latest', mavenSettingsConfig: 'camunda-maven-settings', options: [artifactsPublisher(disabled: true), junitPublisher(disabled: true)]) {
-                runMaven(true, false, false, getMavenProfileDir(env.PROFILE), getMavenProfileCmd(env.PROFILE) + getDbProfiles(env.DB) + " " + getDbExtras(env.DB), true)
+                runMaven(true, false, false, 'engine/', 'clean test -P' + getDbProfiles(env.DB) + " " + getDbExtras(env.DB), true)
```

Comment:
These changes should be reverted, since they remove the `webapps-*` and `engine-unit-authorizations` tests.